blocks/procedures.js

```diff
@@ -468,13 +453,28 @@ Blockly.Blocks['procedures_mutatorcontainer'] = {
   }
 };
 
+var parameterCounter = 1;
```

Comment:
Using a global variable like this is going to create problems. Instead you'll want to do something similar to how new procedure names are generated, but just scanning the blocks in the mutator workspace. See [findLegalName](https://github.com/google/blockly/blob/7679d227df8f62a12316334ee708d4a9a800ef17/core/procedures.js#L89).